shared/src/commonMain/kotlin/co/nimblehq/avishek/phong/kmmic/data/remote/HttpRequestBuilder.kt

```diff
@@ -0,0 +1,24 @@
+package co.nimblehq.avishek.phong.kmmic.data.remote
+
+import io.ktor.client.request.*
+import kotlinx.serialization.SerializationException
+import kotlinx.serialization.json.Json
+import kotlinx.serialization.json.encodeToJsonElement
+import kotlinx.serialization.json.jsonObject
+import co.nimblehq.avishek.phong.kmmic.BuildKonfig
+import co.nimblehq.avishek.phong.kmmic.domain.model.AppError
+
+fun HttpRequestBuilder.path(path: String) {
+    url(BuildKonfig.BASE_URL.plus(path))
+}
```

Comment:
Optional: expression body for one-liner looks nicer.
```suggestion
fun HttpRequestBuilder.path(path: String) = url(BuildKonfig.BASE_URL.plus(path))
```